OasysGH/Units/Helpers/FilteredUnits.cs

```diff
@@ -35,11 +35,11 @@ public class FilteredUnits {
       AreaUnit.SquareFoot.ToString()
     });
 
-    public static List<string> FilteredAxialStiffnessUnits = new List<string>(Enum.GetNames(typeof(AxialStiffnessUnit)).Skip(1).ToList());
+    public static List<string> FilteredAxialStiffnessUnits = new List<string>(Enum.GetNames(typeof(AxialStiffnessUnit)).ToList());
```

Comment:
why were we even skipped one?

Reply:
Even I was struggling to understand why? Those part of codes gives whole list of units available under particular unit type and Default unit form was populated with intended unit. However, skipping one item from list resulted in undesired behaviour as when user changed UI unit to FPS, It requested/expected monent to be KilopoundForceFoot but that was missing from list and combobox failed to set text

Reply:
Previously enum id were different than current one and somehow worked